client/src/utils/tone/index.ts

```diff
@@ -0,0 +1,55 @@
+import store from 'store';
+import { FlashMessages } from '../../components/Flash/redux';
+import { Themes } from '../../components/settings/theme';
+
+const toneUrls = {
+  tryAgain: 'https://campfire-mode.freecodecamp.org/try-again.mp3',
+  cert: 'https://campfire-mode.freecodecamp.org/cert.mp3',
+  day: 'https://campfire-mode.freecodecamp.org/day.mp3',
+  night: 'https://campfire-mode.freecodecamp.org/night.mp3',
+  donate: 'https://campfire-mode.freecodecamp.org/donate.mp3',
+  chalComp: 'https://campfire-mode.freecodecamp.org/chal-comp.mp3',
+  guitarChordOne: 'https://campfire-mode.freecodecamp.org/guitar_chord1.mp3'
```

Comment:
```suggestion
  guitarChordOne: 'https://tonejs.github.io/audio/berklee/guitar_chord1.mp3'
```
This one is from their FOSS library - we don't have a custom sound for this yet.

Reply:
Wow. Nice catch. I wondered why it was not kebab-case